package.json

```diff
@@ -97,7 +97,7 @@
   },
   "snyk": true,
   "dependencies": {
-    "@uswds/compile": "1.2.1",
+    "@uswds/compile": "github:uswds/uswds-compile#dw-gulp-sass-update",
```

Comment:
I'm using a feature branch here where I fixed the Sass JS API deprecation issues. We can link it to a formal compile release when that exists